catroid/src/main/java/org/catrobat/catroid/utils/CrashReporter.java

```diff
@@ -44,10 +42,8 @@
 
 	private static final String TAG = CrashReporter.class.getSimpleName();
 
-	public static final String RECOVERED_FROM_CRASH = "RECOVERED_FROM_CRASH";
-	public static final String EXCEPTION_FOR_REPORT = "EXCEPTION_FOR_REPORT";
-
 	private static SharedPreferences preferences;
+	public static final String EXCEPTION_FOR_REPORT = "EXCEPTION_FOR_REPORT";
 	private static boolean isCrashReportEnabled = BuildConfig.CRASHLYTICS_CRASH_REPORT_ENABLED;
```

Comment:
For making a variable upper case it needs to be a final (and for obvious reasons checkstyle showed the warnings). Now we cannot set it to upper case as this variable can't be set to final and we are changing its value for [the testing purpose](https://github.com/nishantt95/Catroid/blob/7a78857c1d88f12c5260f071abb5663c781b130f/catroid/src/main/java/org/catrobat/catroid/utils/CrashReporter.java#L76-L79).